flutter/shell/platform/tizen/tizen_window_elementary.cc

```diff
@@ -403,4 +444,74 @@ void TizenWindowElementary::PrepareInputMethod() {
       [this](std::string str) { view_delegate_->OnCommit(str); });
 }
 
+#ifdef COMMON_PROFILE
+
+int32_t TizenWindowElementary::GetExternalOutputId() {
+  eom_output_id* output_ids = NULL;
+  eom_output_id output_id = 0;
+  eom_output_type_e output_type = EOM_OUTPUT_TYPE_UNKNOWN;
+  int32_t id_cnt = 0;
+
+  output_ids = eom_get_eom_output_ids(&id_cnt);
+  if (id_cnt == 0) {
+    FT_LOG(Error) << "No external outputs supported";
```

Comment:
Always end a sentence with a period.

```suggestion
  if (!output_ids || id_cnt == 0) {
    FT_LOG(Error) << "No external output found.";
```

Reply:
done